index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Hello Microverse Project</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="header">
+    <div>
```

Comment:
[Optional] This div is not necessary, as you can see in the first line of [HTML and CSS best practices guideline](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), as the focus of this project was not the HTML and CSS it won't block the project approval, but writing clean, semantic code is gonna give you confidence in future projects.